project/plugins.sbt

```diff
@@ -29,3 +29,5 @@ addSbtPlugin("pl.project13.scala" % "sbt-jmh" % "0.1.6")
 libraryDependencies += "com.timgroup" % "java-statsd-client" % "2.0.0"
 
 addSbtPlugin("com.typesafe.sbt" % "sbt-native-packager" % "1.0.0-RC1")
+
+addSbtPlugin("com.timushev.sbt" % "sbt-updates" % "0.1.8")
```

Comment:
Why not? If something goes wrong we can just remove it. We can make it a habit to run it every once in a while.